### PR TITLE
Fixed pre/postmultrotate methods

### PR DESCRIPTION
**Pull request by jtorresfabra**

### CODE REVIEW

File: sources/osg/Matrix.js [hunk 1]
```diff
@@ -652,9 +653,9 @@ var Matrix = {
     postMultRotate: ( function () {
         var r = matrixCreate();
         return function ( matrix, quat ) {
-            Matrix.makeRotateFromQuat( r, quat );
+            Matrix.makeRotateFromQuat( quat, r );
             Matrix.postMult( matrix, r );
-            return matrix;
+            return r;
```
Comment (stephomi) [inline sources/osg/Matrix.js]: r is a local temporary variable, shouldn't it be Matrix.postMult(r, matrix) ?

Comment (jtorresfabra) [inline sources/osg/Matrix.js]: humm..Sorry I'm probably a bit slow today,  I don't get it:
The result of doing premult( a, b ) should be equal to postmult( b, a ). Isn't it? 
So if I do what you say,( postmult( r, matrix ) ) then is the same as premult(matrix, r ), which is not the intended way. 

What I'm missing?

Comment (stephomi) [inline sources/osg/Matrix.js]: preMult( a, b ) -> a = a * b
postMult( b, a ) -> a = b * a
The "pre" and "post" simply tell where the result will be stored

Comment (jtorresfabra) [inline sources/osg/Matrix.js]: So postmult(a,b) gives the same result as premult(a,b) ? The only thing that changes is the variable where you store the result? 

This is a bit confusing. 
Tomorrow I will review the maths. Thanks for your comments :+1:

Comment (stephomi) [inline sources/osg/Matrix.js]: > So postmult(a,b) gives the same result as premult(a,b) ? The only thing that changes is the variable where you store the result?

Yes that's it.
You can use mult(a, b, result), which is not confusing and result can be either a or b.

Comment (jtorresfabra) [inline sources/osg/Matrix.js]: I'm gonna test now, but matrices don't have conmutative property, so if what you say is true, then is somethign too wrong in osgjs.

Comment (jtorresfabra) [inline sources/osg/Matrix.js]: Ok so what you say is true, but the maths in osgjs are wrong in my opinion.

Comment (jtorresfabra) [inline sources/osg/Matrix.js]: @stephomi changed the PR to follow osgjs convention. I talked whith @cedricpinson  and all is clear.

**Pull request merged**